packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -67,6 +67,8 @@ import {
 	sequenceIntervalHelpers,
 	startReferenceSlidingPreference,
 } from "./intervals/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { SequenceIntervalClass } from "./intervals/sequenceInterval.js";
```

Comment:
What is preventing this from following preferred structure and being in ./intervals/index.js?
If keeping, please comment on the lint disable.

Reply:
x several more instances of this.

Reply:
sure. this isn't meant to be checkin ready. hence the draft